### DISCUSSION TITLE
how can I translate the "all" in the filter bar (Portfolio widget > filter bar > All)

### DISCUSSION THREAD

**Discussion by maxnes-des** [elementor/elementor#14443] [Developers Community]
Hello everybody. 

I am using the portfolio widget on my German side and I want to change in the filter bar the English "All" to the German "Alle".

is this possible with custom css? 

In the WP general settings German is selected and the translation are also update.

![Bildschirmfoto 2021-04-06 um 11 50 54](https://user-images.githubusercontent.com/73482486/113693105-898bc500-96ce-11eb-8a10-110f828e63f0.png)

**Comment by 4bluedog** [comment]
Hi,

Just add custom css:

`li[data-filter="__all"] {
  visibility: hidden;
}
li[data-filter="__all"]:after {
  content: 'Wszystkie';
  visibility: visible;
}`

**Reply by dennis2454Ps** [reply]
gracias por tu aporte
